packages/core/src/util/requestUtils.ts

```
@@ -0,0 +1,200 @@
+ *
+ * @param url URL to get the data from.
+ * @param onload Optional function to execute for a successful response.
+ * @param onerror Optional function to execute on error.
+ * @param binary Optional boolean parameter that specifies if the request is binary.
+ * @param timeout Optional timeout in ms before calling ontimeout.
+ * @param ontimeout Optional function to execute on timeout.
+ * @param headers Optional with headers, eg. {'Authorization': 'token xyz'}
+ */
+export const get = (
+  url: string,
+  onload: Function | null = null,
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)